rustic-babel.el

```diff
@@ -9,6 +9,8 @@
 (require 'ob-ref)
 (require 'ob-core)
 
+(require 'inheritenv)
```

Comment:
I'm afraid we can't just assume the package is present. Do you think we can use this functionality only when inheritenv is present ? I'm hesitant to make this an additional dependency, it seems it isn't used this way by many packages https://melpa.org/#/inheritenv

Reply:
@brotzeit That's a good point!  I pushed up a new commit which only uses inheritenv if available.  How does this seem to you now?